crates/aptos-faucet/core/src/server/run.rs

```diff
@@ -212,11 +211,6 @@ impl RunConfig {
             }));
         }
 
-        println!(
-            "Faucet is running. Faucet endpoint: http://{}:{}",
-            self.server_config.listen_address, self.server_config.listen_port
-        );
```

Comment:
So, this no longer prints out for if you're running a faucet on your own?

Reply:
It does, I just moved this into the healthchecking logic of the CLI.